kuma/wiki/templates/wiki/document.html

```diff
@@ -399,6 +416,15 @@ <h2 class="offscreen">{{ _('Document Tags and Contributors') }}</h2>
     <menuitem data-action="$edit" label="{{_('Edit page')}}"></menuitem>
     <menuitem data-action="$history" label="{{_('View page history')}}"></menuitem>
   </menu>
+
+  <!-- embed scrollback widget based on the tags on the page -->
+  {% set tags = document.tags.all() %}
+  {% for tag in tags %}
+    {% if settings.SCROLLBACK_ROOM_MAP[tag.name] and waffle.flag('scrollback_widget') %}
+      <button class = "scrollback-button" onclick='addScrollback("{{settings.SCROLLBACK_ROOM_MAP[tag.name]}}")'> Chat about {{tag.name}} </button>
+    {% endif %}
+  {% endfor %}
+
```

Comment:
@jezdez 
> I don't understand this block, why do we have to go through the list of document tags if the setting for this only contains one item?

The settings can have more tag to room mappings in the future. 

>  The call to settings.SCROLLBACK_ROOM_MAP[tag.name] will also raise a KeyError as soon as one of the tags don't match

If I'm not mistaken, this approach works fine with Jinja2 templates. Is there a better way to check if a key exists in a dict in Jinja2?